src/assets/data/localisations.ts

```diff
@@ -302,6 +302,16 @@ export const Localisations = {
     pl: 'Data rejestracji',
     hb: 'תאריך הרשמה'
   },
+  SETTINGS_DAYS_FROM_ENROL: {
+    da: 'Dage fra registrering',
+    de: 'Tage ab Registrierung',
+    en: 'Days From Enrolment',
```

Comment:
Probably better to call this "Days Since Enrolment". But this is also fine if you prefer this.

Reply:
Yes actually that sounds better. I've made the changes.